docs/networking.md

```diff
@@ -91,3 +91,14 @@ public (active)
   icmp-blocks: 
   rich rules:
 ```
+
+Getting started with firewalld:
+```sh
+# Add podCIDR and serviceCIDR IPv4 ranges
+$ firewall-cmd --add-source=10.244.0.0/16 --add-source=10.96.0.0/12
```

Comment:
> Are you certain we need to add both sources? I haven't checked it but from the top of my head this shouldn't be necessary

Ok, this might mean that the existing firewalld `--list-all` example could be cleaned up as well. Any clue how to test the "vip" services?
I hope I can run some experiments later this week.